src/app/modules/account/pages/child-info/child-info.component.html

```diff
@@ -92,12 +92,7 @@ <h2><strong>Update Child #{{updatedChildren.length - idx}}</strong></h2>
 
       <!-- Add/Remove/Update Buttons-->
       <div *ngIf="hasChild">
-        <p class="horizontal-line">
```

Comment:
The issue that Kai raised seems to only refer to the Spouse Info page.

Reply:
Thanks. Added it back.